L03-generics/src/main/java/ru/otus/Main.java

```diff
@@ -0,0 +1,35 @@
+package ru.otus;
+
+import java.util.*;
+import java.util.Comparator;
+
+public class Main {
+    public static void main(String[] args) {
+        DIYArrayList<Integer> MyArray = new DIYArrayList<Integer>();
```

Comment:
Переменные принято писать в camelCase

```suggestion
        DIYArrayList<Integer> myArray = new DIYArrayList<Integer>();
```